### ISSUE TITLE
Add relevant pages to benefits

### ISSUE THREAD

**Issue by chartgerink** [ubvu/open-handbook#41]
> One thing that I think would be good when I see this is if the little tiles that sum up the benefits of using the resource could be interactive/clickable. For example, the 'collaborative' tile could refer people to the contributing page, and Quality control could go to a page where we explain that process.
> I'm not sure if this is indeed the plan but doesn't work yet because it's only a preview; or if this isn't possible. It's just that I, as a user, would expect this to be clickable (also because the VU website has clickable tiles that look kind of similar).

Originally from @Elisa-on-GitHub in #35.

**Comment by Karvovskaya** [comment]
Yes, exactly! We were thinking that it might be nice to make the tiles clickable. At least VU pages work these way and we are already trained to click on any tile we see.

**Comment by chartgerink** [comment]
How would we want to map the tiles to the pages?

Collaborative --> Contributing
Responsive --> ???
Open source --> GitHub repo
Quality control --> ???
Local --> ???
Flexible --> ???

I figure if we make one clickable, they all should be. Happy to implement this once we have them mapped 👍
